### PR TITLE
Add test for FATE319471 (authorized_keys handling)

### PR DESCRIPTION
**Pull request by imobachgs**
Test for FATE#319471 (PR https://github.com/yast/yast-users/pull/113).

### CODE REVIEW

File: aytests/sles12.xml [hunk 1]
```diff
@@ -826,6 +827,20 @@ ls -l /mnt/$BASE
...
     <user>
       <encrypted config:type="boolean">false</encrypted>
+      <fullname>suse</fullname>
+      <gid>100</gid>
+      <home>/home/suse</home>
+      <shell>/bin/bash</shell>
+      <uid>1001</uid>
+      <user_password>nots3cr3t</user_password>
+      <username>suse</username>
+      <authorized_keys config:type="list">
```
Comment (schubi2) [inline aytests/sles12.xml:837]: Interesting, I have never heard about before :-) Now I am understanding why you would like to adapt the official docu too.

**Comment by schubi2** [comment]
LGTM

File: aytests/sles12.xml (hunk 1 above)
Comment (imobachgs) [reply aytests/sles12.xml:837]: That's a new feature :-) (implemented [here](https://github.com/yast/yast-users/pull/113))

**Comment by imobachgs** [comment]
@schubi2 Thanks for the review. I'll merge the test as soon as the new yast2-schema is included.

**Comment by schubi2** [comment]
LGTM again :-)

**Comment by imobachgs** [comment]
Thanks!

**Pull request merged**